tsfresh/__init__.py

```diff
@@ -18,6 +18,18 @@
     __version__ = 'unknown'
 
 
+# Set default logging handler to avoid "No handler found" warnings.
+import logging
+try:  # Python 2.7+
+    from logging import NullHandler
```

Comment:
I agree with @MaxBenChrist that the problem are these multiple no-handlers-found messages.